src/main/java/core/basesyntax/DataConverterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
```

Comment:
According to the checklist, the `DataConverterImpl` class should be placed in the `service.impl` package to organize the code better.